<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(17, 122, 102);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_animuax">
       <attribute name="title">
        <string>Gestion des animaux</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>1282</width>
          <height>861</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>411</y>
           <width>281</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Recherche</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Ajout_Button_2">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>320</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>50</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Gestion des animaux</string>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>90</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>690</x>
           <y>170</y>
           <width>331</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Date de naissance :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="Date_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);

</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>250</y>
           <width>331</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Numero cage :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NumCage_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>NÂ° Cage</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_7">
         <property name="geometry">
          <rect>
           <x>690</x>
           <y>250</y>
           <width>331</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Nourriture :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Nourriture_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <item>
             <property name="text">
              <string>Carnivore</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Herbivores</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Omnivores</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_8">
         <property name="geometry">
          <rect>
           <x>291</x>
           <y>171</y>
           <width>331</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Type Animal :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Type_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);

</string>
            </property>
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="splitter_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>1261</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QTableView" name="tab_animaux_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>440</y>
           <width>1211</width>
           <height>351</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(175, 175, 175);</string>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter_5">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>800</y>
           <width>431</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit_Edit_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="Edit_Button_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
          <widget class="QPushButton" name="Del_Button_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="Genpdf">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>800</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Generer PDF</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_map">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <widget class="QGraphicsView" name="Map">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>1241</width>
          <height>731</height>
         </rect>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>19</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>NumÂ° Cage :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="numCage">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="RechCage">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>20</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <widget class="QListWidget" name="infoList">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>1261</width>
          <height>631</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(175, 175, 175);</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Animal :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="LineEdit_Info">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Gestion de la nouriture</string>
       </attribute>
       <widget class="QTableView" name="Tab_repas">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>1221</width>
          <height>681</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(175, 175, 175);</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_9">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>801</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>ID de l'animal :</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_id">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="AjouterRepas">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Ajouter repas</string>
         </property>
        </widget>
        <widget class="QPushButton" name="SuppRepas">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Supprimer repas</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
